webapp/channels/src/sass/utils/_flex.scss

```diff
@@ -18,6 +18,10 @@
     -ms-flex-preferred-size: 0;
 }
 
+.flex-auto {
```

Comment:
Unused?

Reply:
This is used in the `ChannelInviteModal` file on master.